### ISSUE TITLE
Syntax error in function returning object literal:

### ISSUE THREAD

**Issue by jonathanhardie** [TiVo/intellij-haxe#117]
function foo() {
		return {
			bar: "hello",
			baz: "world"
		};
	}

![return-obj-literal](https://cloud.githubusercontent.com/assets/477179/6014735/c29bb5d6-abce-11e4-88c6-3be5bcda4191.png)

**Comment by EBatTiVo** [comment]
Yep, this is broken in build 0.8.1.1.TiVo.ClassHierarchy.11.

Bar is an unresolved symbol, and the colon is unexpected.
```    bar: "hello",```

**Comment by as3boyan** [comment]
Yeah, it might be treated as just block of code, not the object literal, object literal should be have priority than block of code...

**Comment by eliasku** [comment]
@EBatTiVo @jonathanhardie 
Fixed?
![image](https://cloud.githubusercontent.com/assets/3038174/13174461/dbd3e39e-d714-11e5-8171-a6154693bf82.png)
